lib/helpers/storage.ts

```diff
@@ -236,3 +249,13 @@ export const multerImageFileFilter: MulterFileFilter = (
     return cb(new BadRequestException(ImageFileErrors.INVALID_FORMAT), false);
   }
 };
+
+
+function isValidFileName(fileName: string): boolean {
```

Comment:
@mayank2424 Can this function be made exportable like the others in this file? Maybe add regex as second parameter to make it more re-usable?

Reply:
@mayank2424 Please let me know once this is done. I will release a new version which can be re-used in the project.

Reply:
@prateekkathal Updated the changes!